tests/savepoint/conftest.py

```diff
@@ -481,30 +481,6 @@ def get_communicator(comm, layout):
     return communicator
 
 
-def pytest_addoption(parser):
-    parser.addoption("--which_modules", action="store")
-    parser.addoption("--skip_modules", action="store")
-    parser.addoption("--print_failures", action="store_true")
-    parser.addoption("--failure_stride", action="store", default=1)
-    parser.addoption("--data_path", action="store", default="./")
-    parser.addoption("--python_regression", action="store_true")
-    parser.addoption("--threshold_overrides_file", action="store", default=None)
```

Comment:
Your description indicates " command-line arguments were not properly registered." Why would that be the case, shouldn't this contest be independent of the ones in tests/conftest.py?

Reply:
Ah I see it's the 'parent' conftest, not the unit one

Reply:
I misunderstood how conftest files work, basically the tests are only allowed to have definitions of `pytest_addoption` and `pytest_configure` in one conftest file in the root of the test directory. It shouldn't break anything, since these are all optional arguments/markers. Without this change, none of the flags are recognized anymore.